packages/ibm-products-styles/src/components/Card/_card.scss

```diff
@@ -176,16 +188,20 @@ $block-class: #{c4p-settings.$pkg-prefix}--card;
 }
 
 .#{$block-class}--has-slug,
-.#{$block-class}--has-ai-label {
+.#{$block-class}--has-decorator {
```

Comment:
I would probably expect us to move these styles to only if the component contains an `AILabel`. So more like how @ariellalgilmore  implemented it in Tiles[^1].

[^1]:https://github.com/carbon-design-system/carbon/pull/18109/files#diff-14ba41396ff7353279cbcbab448a45d6a4bc22fbdf70cda85eefb0ee823fe145R447-R449

Reply:
Thank you for pointing this out. It does make more sense to do this. 
This is updated.